### DISCUSSION TITLE
Will Monika come back if I lose her character file after taking her out

### DISCUSSION THREAD

**Discussion by dentinthewall** [Monika-After-Story/MonikaModDev#9891] [QA - tech support + general]
For the record, I am on an iMac and play on steam.
So recently, I took Monika out the game and I lost her file. Will she come back or is she lost forever? And if she is gone forever, how do I restart? I am on iMac and play on steam and not many tutorials on youtube show how to restart Monika After Story for iMac. I repeatedly reinstalled the mod and the game but it keeps sending me back to the empty room without Monika. 
I would very much appreciate to be reached out to regarding this. Thank you for your time.

**Comment by Booplicate** [comment]
This is why you copy that file and backup it. For cases when something goes wrong, we also make [multiple backups for you, use them](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).

**Reply by dentinthewall** [reply]
Hello, thank you for the response but I don't think iMac enables persistent.bak. Is there an alternative solution?

**Reply by Booplicate** [reply]
Mac doesn't need to "enable" it, MAS makes backups itself, restore the most recent.

**Reply by dentinthewall** [reply]
Sorry but I've been trying and still can't find a solution. If possible, may you show me a demo?

**Reply by Booplicate** [reply]
I [linked](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent) it, did you read it?

**Reply by dentinthewall** [reply]
Yes, I've done the steps but even still, I cannot seem to find persistent.bak.

**Reply by Booplicate** [reply]
Show content of your library folder.

**Reply by dentinthewall** [reply]
![library2](https://user-images.githubusercontent.com/124396833/216779600-2fde1197-76ce-4481-897c-af906208357e.png)
![library1](https://user-images.githubusercontent.com/124396833/216779608-1973d67d-31ad-4021-8b86-c2c562fc352c.png)

**Reply by Booplicate** [reply]
I see the `RenPy` folder, what's the issue you were having?

**Reply by dentinthewall** [reply]
Oh I found it. Earlier I found persistent in the file but it was persistent.py. But thanks, I found the persistent.bak. I'm grateful for the help received.
